### PR TITLE
Added db id migration script

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: migration/db-id/mssql/migration.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  curl "$(get_db_id_migration_path)/migration/db-id/mssql/entrypoint.sh" --output ${migration_entrypoint}
+  docker-compose -f docker-compose-mssql-polling.yml -f ${migration_tool} up --build --no-deps mssql-migration
+  docker-compose -f docker-compose-mssql-polling.yml -f ${migration_tool} stop mssql-migration
+  docker-compose -f docker-compose-mssql-polling.yml -f ${migration_tool} rm -f mssql-migration
+
+  rm -rf migration
+  rm -rf ${migration_tool}
+
+else
+  echo "Unknown Database"
```
Comment (cer) [inline migration/db-id/mssql/migration.sh:33]: The error message should include `$DATABASE`

File: migration/db-id/mssql/migration.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+set -e
+
+get_db_id_migration_path () {
+  search="eventuateCommonVersion="
+  version_line="$(grep $search ./gradle.properties)"
+  version=${version_line#$search}
+  echo "${db_id_migration_repository}/${version}"
```
Comment (cer) [inline migration/db-id/mssql/migration.sh:9]: This should be uppercase and optional, e.g. default to this repo.

File: migration/db-id/mssql/migration.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
```
Comment (cer) [inline migration/db-id/mssql/migration.sh:1]: This file should not be in the mssql directory. it should be one level up

**Review by cer — CHANGES_REQUESTED** [review]

File: migration/db-id/mssql/migration.sh (hunk 1 above)
Comment (dartartem) [reply migration/db-id/mssql/migration.sh:33]: ok

File: migration/db-id/mssql/migration.sh (hunk 2 above)
Comment (dartartem) [reply migration/db-id/mssql/migration.sh:9]: ok

File: migration/db-id/mssql/migration.sh (hunk 3 above)
Comment (dartartem) [reply migration/db-id/mssql/migration.sh:1]: ok

**Comment by dartartem** [comment]
Closed because of https://github.com/eventuate-foundation/eventuate-common/pull/77

**Pull request closed without merging**